session.go

```diff
@@ -2254,29 +2254,46 @@ type traceWriter struct {
 	session *Session
 	w       io.Writer
 	mu      sync.Mutex
+
+	tracingInfoFetchAttempts int
+	tracingInfoFetchInterval time.Duration
 }
 
 // NewTraceWriter returns a simple Tracer implementation that outputs
 // the event log in a textual format.
 func NewTraceWriter(session *Session, w io.Writer) Tracer {
-	return &traceWriter{session: session, w: w}
+	return &traceWriter{session: session, w: w, tracingInfoFetchAttempts: 5, tracingInfoFetchInterval: 3 * time.Millisecond}
```

Comment:
Please make them configurable.

Reply:
I am not sure if I should change the `NewTraceWriter` arguments. It can be a breaking change for users I guess. If someone wants to use values different than default they could just use `&traceWriter{...}` instead of `NewTraceWriter`, WDYT?

Reply:
SetMaxAttempts
SetSleepInterval

Please also change their names, it does not make sense to have them so long in the context of tracer